content/influxdb/v2.2/write-data/troubleshoot.md

```diff
@@ -60,25 +77,240 @@ Write requests return the following status codes:
+If you try to write this data to a bucket that has the [`explicit` schema type](/influxdb/cloud/organizations/buckets/bucket-schema/) and doesn't have an `airSensors` schema, the `/api/v2/write` InfluxDB API returns an error and the following data:
+
+```json
+{
+  "code": "invalid",
+  "message": "3 out of 3 points rejected (check rejected_points in your _monitoring bucket for further information)"
+}
+```
+
+InfluxDB logs three `rejected_points` entries, one for each field.
+
+| _measurement    | _field | _value | field       | measurement | reason                            |
+|:----------------|:-------|:-------|:------------|:------------|:----------------------------------|
+| rejected_points | count  | 1      | humidity    | airSensors  | measurement not allowed by schema |
+| rejected_points | count  | 1      | co          | airSensors  | measurement not allowed by schema |
+| rejected_points | count  | 1      | temperature | airSensors  | measurement not allowed by schema |
+
+##### Detect a field type mismatch
+
+InfluxDB rejects a point if the [measurement](/influxdb/cloud/reference/key-concepts/data-elements/#measurement) matches the **name** of a bucket schema and the field data types don't match.
+The `rejected_points` entry contains the following reason:
+
+| Reason                              | Meaning                                                                                              |
+|:------------------------------------|:-----------------------------------------------------------------------------------------------------|
+| `field type mismatch with schema`   | The point has the same measurement as a configured schema and they have different field value types. |
+
+Consider a bucket that has the following `airSensors` [`explicit bucket schema`](/influxdb/cloud/organizations/buckets/bucket-schema/):
+
+```json
+{
+ "name": "airSensors",
+ "columns": [
+   {
+     "name": "time",
+     "type": "timestamp"
+   },
+   {
+     "name": "sensorId",
+     "type": "tag"
+   },
+   {
+     "name": "temperature",
+     "type": "field",
+     "dataType": "float"
+   },
+   {
+     "name": "humidity",
+     "type": "field",
+     "dataType": "float"
+   },
+   {
+     "name": "co",
+     "type": "field",
+     "dataType": "float"
+   }
+ ]
+}
+```
+
+The following [line protocol](/influxdb/cloud/reference/syntax/line-protocol/) data has an `airSensors` measurement, a `sensorId` tag, and three fields (`temperature`, `humidity`, and `co`).
+
+```
+airSensors,sensorId=L1 temperature=90.5,humidity=70.0,co=0.2 1637014074
+airSensors,sensorId=L1 temperature="90.5",humidity=70.0,co=0.2 1637014074
+```
+
+In the example data, one of the points has a `temperature` field value with the _string_ data type.
+However, the `airSensors` schema requires `temperature` to have the _float_ data type.
+If you try to write the example data to the `airSensors` bucket schema,
+InfluxDB returns a `400` error and a message that describes the result:
```

Comment:
```suggestion
Because the `airSensors` schema requires `temperature` to have the _float_ data type, 
InfluxDB returns a `400` error and a message that describes the result:
```